src/plugins/elasticsearch/lib/health_check.js

```diff
@@ -65,7 +65,7 @@ module.exports = function (plugin, server) {
     if (timeoutId) return;
 
     var myId = setTimeout(function () {
-      check().finally(function () {
+      Promise.try(check).finally(function () {
```

Comment:
So that we can use bluebird's `.finally()` method. the es client used to expose bluebird but to prevent applications from writing code bound to an implementation detail (like this) esjs moved to an es6 promise solution.